### PR TITLE
Adds durathread.

### PR DESCRIPTION
**Pull request by lolman360**
## About The Pull Request

Adds duathread and durathread items. Also gives botany a worse way to get cloth from cotton.
From /tg/station PRs:

https://github.com/tgstation/tgstation/pull/40563
https://github.com/tgstation/tgstation/pull/45261/
https://github.com/tgstation/tgstation/pull/45820

Balance changes:
Durathread beret and beanie offer same defence as helmet but are more expensive. 
Durathread bandolier can carry any obj/item/ammo_casing. Want to handload L6 SAW mags with your bandolier? Totally doable.

## Why It's Good For The Game

New stuff = cool. Also POWERCREEP.

## Changelog
:cl:
add: NT has authorized shipments or Cotton to Megaseed Servitors. It's time to start picking, liggers.
/:cl:

### CODE REVIEW

**Comment by lolman360** [comment]
SKREEEEEEEE MERGE CONFLICTS

**Comment by lolman360** [comment]
why does this still fucking conflict

**Comment by lolman360** [comment]
@kevinz000 help
so I downloaded Cit's Master branch, deleted my hats.dmi, replaced it with the cit hats.dmi and then added my 3 sprites to the new cit hats.dmi.
it STILL CONFLICTS

**Comment by lolman360** [comment]
consulted sishen and blackmajor on discord and apparently conflict is a meme

**Comment by Linzolle** [comment]
Jumpsuit and bandana need digitigrade and snout variations respectively.

**Comment by lolman360** [comment]
fuck liggers lmao

**Comment by Linzolle** [comment]
> 
> 
> fuck liggers lmao

Very constructive, thank you.

**Comment by lolman360** [comment]
i stole these sprites from /tg/ do you really think i 1. want to sprite lizard-friendly versions and 2. want to ask /tg/ to sprite lizard-friendly versions

**Comment by lolman360** [comment]
i literally cant find regular jumpuit digitigrade variant anywhere where am i supposed to reference these stupid sprites

**Comment by Linzolle** [comment]
Try /modular_citadel/icons/mob/uniforms_digi.dmi and muzzled_mask.dmi

**Comment by lolman360** [comment]
done

File: code/modules/crafting/recipes.dm
Comment (Sishen1542) [inline code/modules/crafting/recipes.dm:938]: add in recipes for helmet and vest, should be good to go after that

Comment (lolman360) [reply code/modules/crafting/recipes.dm:938]: done thansks

**Comment by lolman360** [comment]
pogchamp. merge conflict resolved. travis passes.

**Comment by Sishen1542** [comment]
a winner is you

**Comment by lolman360** [comment]
watch this get closed by maintainer because they hate me

**Review by silicons — APPROVED** [review]

**Pull request merged**
